bodhi/server/services/updates.py

```diff
@@ -612,6 +612,12 @@ def new_update(request):
                             pending_signing_tag=pending_signing_tag,
                             from_tag=from_tag,
                             candidate_tag=candidate_tag)
+                        # Set request to testing
+                        # https://github.com/fedora-infra/bodhi/issues/4087
+                        log.debug(f"Setting request for new update {u.alias} from side-tag.")
+                        db = request.db
+                        req = UpdateRequest.testing
+                        u.set_request(db, req, request.user.name)
```

Comment:
I've realized this may not be the best: since the tagging process is made asynchronously by a celery task, we probably end in a situation were the set_request is performed before the celery task tagging process, thus sending a notification on the bus before builds get tagged.

Should I move the set_request action inside the celery task? If so I will need to pass the update alias and the user.name to the task and I'll need to open a database session inside within the celery task.
Is there any other idea on how we can achieve this?